package/libwebsockets/libwebsockets.mk

```diff
@@ -14,9 +14,7 @@ LIBWEBSOCKETS_CONF_OPTS = \
 	-DLWS_WITHOUT_TESTAPPS=ON \
 	-DLWS_IPV6=ON \
 	-DLWS_UNIX_SOCK=ON \
-	-DLWS_WITHOUT_EXTENSIONS=OFF
-LIBWEBSOCKETS_CONF_OPTS += \
```

Comment:
I think we still want `LIBWEBSOCKETS_CONF_OPTS`?

Reply:
This is just removing the redundant call to append to `LIBWEBSOCKETS_CONF_OPTS`. I suppose you could make the case that the rebase onto upstream is easier with a second call to append to `LIBWEBSOCKETS_CONF_OPTS`, but I'm hoping we're able to just use the upstream code on the next sync with the upstream repo.

Reply:
This is a patch trick when tracking upstream - the "second one" is what we added for a customization, so when we patch upstream we can see what was the defaults in the first and then check the second.

Old trick - but - sometimes useful to make redundant for different meaning.

Reply:
I just removed all ChargePoint changes from that list, so no longer relevant.